bitnami/concourse/values.yaml

```diff
@@ -318,6 +329,20 @@ web:
     enabled: false
   ## @param web.configRBAC Set RBAC configuration
   ##
+  ## Configuration for using Conjur as a credential manager.
+  ## Ref: https://concourse-ci.org/conjur-credential-manager.html
+  ##
+  conjur:
+    ## @param web.conjur.enabled Enable the use of Conjur as a credential manager
+    enabled: false
+    ## @param web.conjur.applianceUrl URL of the Conjur instance.
+    applianceUrl: ""
+    ## @param web.conjur.pipelineSecretTemplate Path used to locate pipeline-level secret
+    pipelineSecretTemplate: concourse/{{.Team}}/{{.Pipeline}}/{{.Secret}}
```

Comment:
Is this rendered correctly without quotes, doesn't it give YAML formatting issues?

Reply:
Yes, I checked it:

```
$ kubectl describe pod -concourse-web-XXXXXX-XXXX
...
      CONCOURSE_CONJUR_PIPELINE_SECRET_TEMPLATE:  concourse/{{.Team}}/{{.Pipeline}}/{{.Secret}}
      CONCOURSE_CONJUR_TEAM_SECRET_TEMPLATE:      concourse/{{.Team}}/{{.Secret}}
      CONCOURSE_CONJUR_SECRET_TEMPLATE:           concourse/{{.Secret}}
```